File: app/src/main/java/com/littlefox/app/foxschool/object/result/main/CompanyInformationResult.java
package com.littlefox.app.foxschool.object.result.main;

public class CompanyInformationResult
{
    private String ceo = "";
    private String address = "";
    private String brn = "";
    private String mosrn = "";
    private String ecn_number = "";
    private String phone = "";
    private String fax = "";

    public String getCEO()
    {
        return ceo;
    }

    public String getAddress()
    {
        return address;
    }

    /**
     * 사업자 등록 번호
     * @return 해당 텍스트
     */
    public String getCompanyRegistrationNumber()
    {
        return brn;
    }

    /**
     * 통신 판매업 신고
     * @return 해당 텍스트
     */
    public String getReportCompany()
    {
        return mosrn;
    }

    /**
     * 전자 출판물 인증
     * @return 해당 텍스트
     */
    public String getElectronicPublishingCertification()
    {
        return ecn_number;
    }

    public String getPhoneNumber()
    {
        return phone;
    }

    public String getFaxNumber()
    {
        return fax;
    }
}

File: app/src/main/java/com/littlefox/app/foxschool/object/result/main/CompanyInformationResult.kt
package com.littlefox.app.foxschool.`object`.result.main

import com.google.gson.annotations.SerializedName

class CompanyInformationResult
{
    @SerializedName("ceo")
    private val ceo = ""

    @SerializedName("address")
    private val address = ""

    @SerializedName("brn")
    private val brn = ""

    @SerializedName("mosrn")
    private val mosrn = ""

    /**
     * 전자 출판물 인증
     * @return 해당 텍스트
     */
    @SerializedName("ecn_number")
    private val ecn_number = ""

    @SerializedName("phone")
    private val phone = ""

    @SerializedName("fax")
    private val fax = ""

    fun getCEO() : String = ceo

    fun getAddress() : String = address

    /**
     * 사업자 등록 번호
     * @return 해당 텍스트
     */
    fun getCompanyRegistrationNumber() : String = brn

    /**
     * 통신 판매업 신고
     * @return 해당 텍스트
     */
    fun getReportCompany() : String = mosrn

    fun getElectronicPublishingCertification() : String = ecn_number

    fun getPhoneNumber() : String = phone

    fun getFaxNumber() : String = fax
}